### ISSUE TITLE
published expo-dev-launcher 0.10.4 missing some merged changes

### ISSUE THREAD

**Issue by themichaellai** [expo/expo#16836]
### Summary

Hello! I believe that `expo-dev-launcher` version `0.10.4` may be published incorrectly. More detail below:

I'm running into an issue on Android when building my app locally: 
```
[...]/mobile/node_modules/expo-dev-launcher/android/src/main/AndroidManifest.xml Error:
        Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
```
which appears to be the same issue addressed and fixed in https://github.com/expo/expo/issues/16366. I see on `main` that the fix is there: https://github.com/expo/expo/blob/e1d6b0c241bbbe6256395875614edcec7e5d2cc0/packages/expo-dev-launcher/android/src/main/AndroidManifest.xml#L10 and that it should have been included in `expo-dev-launcher` `0.10.3` (and beyond), but I don't see it in the published package:
```
❯ curl -s https://registry.npmjs.org/expo-dev-launcher/-/expo-dev-launcher-0.10.4.tgz | tar -xz

❮ head -n 12 package/android/src/main/AndroidManifest.xml
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
  package="expo.modules.devlauncher">

  <application>
    <activity
      android:name="expo.modules.devlauncher.launcher.DevLauncherActivity"
      android:screenOrientation="portrait"
      android:theme="@style/Theme.DevLauncher.LauncherActivity"
      android:launchMode="singleTask"
      >
      <intent-filter>
        <action android:name="android.intent.action.VIEW"/>
```

Wondering whether the package was mispublished or whether I'm missing something on my end. Thanks!

### Managed or bare workflow? If you have made manual changes inside of the `ios/` or `android/` directories in your project, the answer is bare!

bare

### What platform(s) does this occur on?

Android

### Package versions

`expo-dev-client` 0.8.4
`expo-updates` 0.11.6
`expo-dev-launcher` 0.10.4

### Environment

```
  expo-env-info 1.0.2 environment info:
    System:
      OS: macOS 12.2.1
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 14.17.5 - ~/.nvm/versions/node/v14.17.5/bin/node
      Yarn: 1.22.11 - ~/.nvm/versions/node/v14.17.5/bin/yarn
      npm: 8.3.2 - ~/.nvm/versions/node/v14.17.5/bin/npm
    Managers:
      CocoaPods: 1.11.0 - /Users/michael/.rvm/gems/ruby-3.0.2/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    IDEs:
      Xcode: 13.2.1/13C100 - /usr/bin/xcodebuild
    npmPackages:
      expo: >=44.0.0-0 <45.0.0 => 44.0.6
      react: 17.0.2 => 17.0.2
      react-native: ^0.66.2 => 0.66.2
    npmGlobalPackages:
      eas-cli: 0.48.2
      expo-cli: 5.3.0
    Expo Workflow: bare
```

### Reproducible demo

Please see the bottom bit of my summary above.

**Comment by matheusmatos** [comment]
I applied this patch to `expo-dev-launcher@0.10.4` and it worked fine - build works with EAS.

```diff
diff --git a/node_modules/expo-dev-launcher/android/src/main/AndroidManifest.xml b/node_modules/expo-dev-launcher/android/src/main/AndroidManifest.xml
index 6634aff..d0ba955 100644
--- a/node_modules/expo-dev-launcher/android/src/main/AndroidManifest.xml
+++ b/node_modules/expo-dev-launcher/android/src/main/AndroidManifest.xml
@@ -7,6 +7,7 @@
       android:screenOrientation="portrait"
       android:theme="@style/Theme.DevLauncher.LauncherActivity"
       android:launchMode="singleTask" 
+      android:exported="false"
       >
       <intent-filter>
         <action android:name="android.intent.action.VIEW"/>
```

I'm not sure why [16366](https://github.com/expo/expo/issues/16366) was written with `android:exported="true"`. This activity needs to be [launchable by other apps](https://developer.android.com/guide/topics/manifest/activity-element#exported)?

But certainly the version published on NPM does not have this change.

**Comment by aindong** [comment]
weird, this is supposed to be fixed and merged with 0.10.3 and 0.10.4 but this versions still doesn't include the
`android:exported="false"` but I see it on the `next (0.11)` version, is this intended?

**Comment by lukmccall** [comment]
Hi everyone, we published a new version `expo-dev-launcher@0.10.5` which includes that fix. This version of dev-launcher is a part of `expo-dev-client@0.8.5`. Sorry for the inconvenience.

**Comment by themichaellai** [comment]
Thank you @lukmccall !
